fe/fe-core/src/main/java/org/apache/doris/analysis/PartitionNames.java

```diff
@@ -48,36 +48,36 @@ public class PartitionNames implements ParseNode, Writable {
     // true if these partitions are temp partitions
     @SerializedName(value = "isTemp")
     private final boolean isTemp;
-    private final boolean allPartitions;
+    private final boolean autoReplace;
```

Comment:
Why do we change the name of this variable? It's kind of misleading with the new name.